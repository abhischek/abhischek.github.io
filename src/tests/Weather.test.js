import { shallow } from 'enzyme';
import React from 'react';
import Weather from '../components/Weather';

it('expect to render Weather component', () => {
	expect(shallow(<Weather />)).toMatchSnapshot();
})