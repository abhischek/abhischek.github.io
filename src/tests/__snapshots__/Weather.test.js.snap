// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Weather component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Weather />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        undefined,
        undefined,
        undefined,
        undefined,
        undefined,
      ],
      "className": "weather__info",
    },
    "ref": null,
    "rendered": Array [
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
        ],
        "className": "weather__info",
      },
      "ref": null,
      "rendered": Array [
        undefined,
        undefined,
        undefined,
        undefined,
        undefined,
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
